README.rst

```diff
@@ -274,12 +274,43 @@ as devices with the following command.
 
    zenbatchload <filename>
 
+Modeling and Monitoring OS Services
+-----------------------------------
+The Linux OS services are modeled using the *zenoss.cmd.linux.os_service* modeler plugin. The following systems are supported:
+
+- systemd (RHEL 7)
+- upstart (RHEL 6)
+- systemV (RHEL 5 and earlier)
+
+Version 2.3.0 and beyond provides support for monitoring the status of the services every collection cycle. However, only **systemd** and **systemV** are supported for monitoring. The following two zProperties filter the services that are to be modeled and thereby monitored.
+
++---------------------------+------------------------------------------+
+| Name                      | Description                              |
++===========================+==========================================+
+| zLinuxServicesModeled     | Accepts regular expressions that         |
+|                           | matches one or more services to model    |
++---------------------------+------------------------------------------+
+| zLinuxServicesNotModeled  | Accepts regular expressions that         |
+|                           | matches one or more services to not model|
++---------------------------+------------------------------------------+
+
```

Comment:
Its probably better to put all the zProperties in a singe table in the configuraiton section, and then mention each of those properties in the sections that refer to them. This way the user has a single place to look at those.
You should also put the default values (if any) in that table as well so users know what is what.

A lot of our docs grow organically and it gets a bit choppy to read them, so this is good reason to consolidate some of this info.

Reply:
Also, not part of your ticket: zIgnoreUnmounted is defined in LM but not mentioned in the docs.

Reply:
As we discussed in our chat, this will remain the same.